deepinv/physics/forward.py

```diff
@@ -741,6 +757,15 @@ def A_dagger(self, y, **kwargs):
 
         return self.V(self.U_adjoint(y) * mask)
 
+    def update_parameters(self, mask=None, **kwargs):
```

Comment:
For the most generic update_parameters, should we make it purely kwargs:
```
def update_parameters(self, **kwargs):
for kwarg in kwargs...
    if hasattr(self, kwarg)...
```
This way, we can call `self.update_parameters(mask=mask)` in `A` without fear that its behaviour will be different when `update_parameters` is overriden.

Reply:
good idea!

Reply:
Can we also replace `if self.mask is None: self.mask = nn.Parameter(...` with `self.update_parameters(mask=mask)` in `DecomposablePhysics.__init__`, `DecomposablePhysics.A`, `A_adjoint`, `A_dagger`?

Reply:
Or I can do this in #242

Reply:
Likewise replacing the `self.filter = ...` in `Blur.__init__`, `Blur.A`...

Reply:
done!